src/compiler/sys.ts

```diff
@@ -1857,12 +1861,19 @@ namespace ts {
             }
 
             function getModifiedTime(path: string) {
+                // Since the error thrown by fs.statSync isn't used, we can avoid collecting a stack trace to improve

```

Comment:
Copied the optimization from `fileSystemEntryExists` as we are using getModifiedTime more often now..